Assets/MixedRealityToolkit.Tests/EditModeTests/Services/TestBaseDataProvider.cs

```diff
@@ -9,13 +9,21 @@ namespace Microsoft.MixedReality.Toolkit.Tests.Services
     /// </summary>
     public class TestBaseDataProvider : BaseDataProvider
     {
+        [System.Obsolete()]
```

Comment:
Can we just clean these up for our test files?

Reply:
yes! i think i missed this one :)